src/main/java/teammates/common/util/EmailLogEntry.java

```diff
@@ -13,13 +13,13 @@
 /** A log entry which contains info about subject, receiver, content and sent date of a sent email*/
 public class EmailLogEntry {
     
+    public String logInfoAsHtml;
```

Comment:
Could this be made private as well? It doesn't seem to be used outside this class